hw/ip/aes/doc/_index.md

```diff
@@ -328,6 +328,13 @@ Future versions of this AES unit thus might employ different means at architectu
 This section discusses how software can interface with the AES unit.
 
 
+## Clear upon Reset
+
+Upon reset, the AES unit will first reseed the internal PRNG for register clearing via EDN, and then clear all key, IV and data registers with pseudo-random data.
+Only after this sequence has finished, the unit becomes idle (indicated in {{< regref "STATUS.IDLE" >}}).
+The AES unit is then ready for software initialization.
+
```

Comment:
maybe also add a comment here that upon idle the registers can no longer be expected to have the value of "resval"

Reply:
Good point, that's now added as well.